tests/FSharp.AWS.DynamoDB.Tests/MetricsCollectorTests.fs

```diff
@@ -139,94 +137,94 @@ type ``Metrics Collector Tests`` (fixture : TableFixture) =
                     cu > 0
                 | _ -> false @>
 
-    let [<Fact>] ``Collect Metrics on Scan`` () =
```

Comment:
note moved to a separate Test Class so those 3 can 
1. share the work of establishing the items
2. run in parallel with the other tests